q2_longitudinal/plugin_setup.py

```diff
@@ -130,6 +130,18 @@
 }
 
 
+formula_description = (
+    'Formulae will be in the format "a ~ b + c", where '
+    '"a" is the metric (dependent variable) and "b" and "c" are '
+    'independent covariates. Use "+" to add a variable; "+ a:b" to '
+    'add an interaction between variables a and b; "*" to include '
+    'a variable and all interactions; and "-" to subtract a '
+    'particular term (e.g., an interaction term). See '
+    'https://patsy.readthedocs.io/en/latest/formulas.html for full '
+    'documentation of valid formula operators. On command line, '
```

Comment:
There are other reasons to enclose it quotes (e.g. `*`) --- maybe this should always recommend wrapping in quotes?

Reply:
👍